wire-runtime/src/jvmMain/kotlin/com/squareup/wire/internal/reflection.kt

```diff
@@ -79,12 +87,14 @@ private fun <M : Message<M, B>, B : Message.Builder<M, B>> getKeys(
 fun <M : Message<M, B>, B : Message.Builder<M, B>> createRuntimeMessageAdapter(
   messageType: Class<M>,
   writeIdentityValues: Boolean,
+  loader: ClassLoader,
```

Comment:
I think we need a default param here as well. For instance, `WireTypeAdapterFactory` is using it